### DISCUSSION TITLE
Any way to prevent JET_IS_DISABLED_MESSAGE from displaying in the console on startup?

### DISCUSSION THREAD

**Discussion by Aquerr** [hazelcast/hazelcast#26222] [Q&A]
As asked in the title. Is it possible to disable the log message telling that the jet engine is disabled?

I mean, I don't need to use Jet Engine and don't need to be notified about it being disabled every time my application is starting.

**Comment by torahmike** [comment]
Yes. 

1. Set hazelcast.logging.type to slf4j and include a slf4j implementation e.g. logback in your classpath
2. In your slf4j implementation's configuration filter , filter out com.hazelcast.system logger
